views/biobank/order.html.twig

```diff
@@ -1,5 +1,5 @@
 {% extends 'base.html.twig' %}
-{% block title %}Participant {{ participant.id }} - {% endblock %}
+{% block title %}Biobank {{ participant.biobankId }} - {% endblock %}
```

Comment:
I would leave the label as "Participant" as the biobank id still identifies a participant.
```suggestion
{% block title %}Participant {{ participant.biobankId }} - {% endblock %}
```